src/explorer/resultsList.ts

```diff
@@ -748,4 +732,62 @@ export class ResultsList {
             caseButton.classList.remove("active");
         }
     }
+
+    /**
+     * Handler to perform keyboard navigation in the results list.
+     * @param keydownEvent Keyboard event
+     */
+    private onResultsListKeyDown(keydownEvent: JQuery.KeyDownEvent): void  {        
+        enum KeyCodes {
+            LeftArrow = 37,
+            UpArrow = 38,
+            RightArrow = 39,
+            DownArrow = 40
+        }
+
+        let direction: 'Previous' | 'Next' | undefined;
+        if (keydownEvent.keyCode === KeyCodes.LeftArrow || keydownEvent.keyCode === KeyCodes.UpArrow) {
+            direction = 'Previous';
```

Comment:
How was it decided that Left Arrow and Right Arrow are also mapped to previous and next? This is inconsistent with common UI patterns.

Reply:
That may be true, but it's good enough for now until your work is done.

---
In reply to: [413164231](https://github.com/microsoft/sarif-vscode-extension/pull/254#discussion_r413164231) [](ancestors = 413164231)